js/preload/preloadMain.js

```diff
@@ -121,6 +123,28 @@ function createAPI() {
         },
 
         /**
+         * Open file in default app.
+         */
+        openFile: function (id) {
+            let fileIndex = _.findIndex(local.downloadItems, {_id: id});
+            let openResponse = remote.shell.openExternal(`file:///${local.downloadItems[fileIndex].filePath}`);
+            if (!openResponse) {
+                remote.dialog.showErrorBox("File not found", 'The file you are trying to open cannot be found in the specified path');
+            }
+        },
+
+        /**
+         * Show downloaded file in explorer or finder.
+         */
+        showInFinder: function (id) {
```

Comment:
why do we need this in the ssf namespace.  this will expose api to JS.  i don't think needed here?